core/tests/puppeteer-acceptance-tests/jasmine.json

```diff
@@ -0,0 +1,10 @@
+{
+  "spec_dir": "core/tests/puppeteer-acceptance-tests/spec",
```

Comment:
Why this path exactly?

Reply:
Now since we are maintaining one single file per user ( which can contain multiple spec/ tests) we would need them under one directory so I went with the name provided by default i.e /spec . We can name it anything if we want.

Reply:
Do we have the proposed folder structure somewhere?

Reply:
Yeah https://docs.google.com/document/d/1_CGvntDKJP3HCBWvigMvlI0Mvl0WaLnOItjBm3x4C7o/edit?usp=drivesdk  but it's outdated as per current structure. As of now we'll not be having multiple individual files for a user, rather a single file with multiple tests. So in total we'll have a handful of test files which can be put under a single directory.

Reply:
Please still update the structure.